rezerwacje-front/src/views/Rooms/components/RoomsForm/RoomsForm.js

```diff
@@ -33,20 +32,17 @@ class Rooms extends React.Component {
     fetch("/api/rooms/all")
       .then((res) => res.json())
       .then((json) => this.setState({ rooms: json }))
-      .catch((e) => alert(e));
+      .catch((e) => console.log(e));
   }
 
   render() {
     return (
-      <div>
-        <Sidebar />
         <div class="main">
```

Comment:
- [ ] Should be className not class
- [ ] main is too broad of a name for a css class.
We need to remember that css is global so if two different people use the same class name like main they will have conflict. As a suggestion it would be good to use styled-components for custom component styles